### PR TITLE
chore: update readme.md for better devex on windows 

### PR DESCRIPTION
**Pull request by irajAamir**
update README for local developers of windows users

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,8 +6,8 @@
   "scripts": {
     "lint": "eslint . --ext .ts,.tsx --fix",
     "clean": "rimraf coverage build tmp",
-    "dev": "nodemon -e 'yml, yaml, js, ts, tsx, json' -x 'ts-node' src/server.ts | bunyan -o simple",
-    "typeorm": "./node_modules/.bin/ts-node ./node_modules/typeorm/cli.js",
+    "dev": "nodemon -e \"yml, yaml, js, ts, tsx, json\" -x \"ts-node\" src/server.ts | bunyan -o simple",
+    "typeorm":"node-require ts-node/register•/node_modules/typeorm/cli.js" ,
```
Comment (hassanazharkhan) [inline package.json:10]:
```suggestion
    "typeorm":"node --require ts-node/register ./node_modules/typeorm/cli.js" ,
```

File: src/config.ts [hunk 2]
```diff
@@ -36,8 +36,8 @@ const conf = convict({
     },
     password: {
       format: '*',
-      default: 'postgres',
-      env: 'DB_PASSWORD',
+      default: 'irajaamir',
```
Comment (hassanazharkhan) [inline src/config.ts:39]: no need to change this!

File: src/config.ts [hunk 3]
```diff
@@ -36,8 +36,8 @@ const conf = convict({
...
+      env: 'irajaamir',
```
Comment (hassanazharkhan) [inline src/config.ts:40]: this as well

File: package.json [hunk 4]
```diff
@@ -29,39 +29,52 @@
...
+    "typescript": "4.5.4"
   },
   "devDependencies": {
     "@types/jest": "27.4.0",
     "ts-jest": "26.5.6",
     "jest": "26.6.3",
     "nodemon": "2.0.15"
-  }
+  },
+  "description": "[![Dependency Status](https://david-dm.org/Microsoft/TypeScript-Node-Starter.svg)](https://david-dm.org/Microsoft/TypeScript-Node-Starter) [![Build Status](https://travis-ci.org/Microsoft/TypeScript-Node-Starter.svg?branch=master)](https://travis-ci.org/Microsoft/TypeScript-Node-Starter)",
```
Comment (hassanazharkhan) [inline package.json:67]: What is this?

**Review by hassanazharkhan — CHANGES_REQUESTED** [review]

File: package.json (hunk 4 above)
Comment (irajAamir) [reply package.json:67]: don't know

**Review by hassanazharkhan — APPROVED** [review]

**Pull request merged**